modules/doctor/breaking.go

```diff
@@ -66,4 +89,11 @@ func init() {
 		Run:       checkUserEmail,
 		Priority:  9,
 	})
+	Register(&Check{
+		Title:     "Check if users have a valid username",
+		Name:      "check-user-name",
```

Comment:
Given that you don't check for a single user but for all instead, what about using
```suggestion
		Name:      "check-user-names",
```
instead?